_pages/about-us/teams/oa

```diff
@@ -1,8 +1,8 @@
 ---
-title: Acquisition (aka AcqStack)
+title: Office of Acquisition
 ---
 
-Welcome to Acqstack’s team page!
+Welcome to OA’s team page!
```

Comment:
This may just be my personal aversion to "welcome to" messages, but I'd strike the sentence. I don't think it adds value.

Reply:
You can strike that sentence.